.loco/config/mysql-common/my.cnf.php

```diff
@@ -37,6 +37,7 @@ function matchVer($pat) {return (bool) preg_match($pat, ver());}
 <?php if (matchVer('/Ver 5.6/')) { ?>
 innodb_large_prefix = 1
 <?php } ?>
+<?php if (!matchVer('/Ver 8.0/')) { ?>innodb_file_format = Barracuda<?php } ?>
```

Comment:
I like how much easier that is now. :)

Small revision: https://github.com/civicrm/civicrm-buildkit/pull/528